Lombiq.HelpfulLibraries/Libraries/Utilities/EnumerableExtensions.cs

```diff
@@ -36,6 +36,21 @@ public static class EnumerableExtensions
             return results;
         }
 
+        /// <summary>
+        /// Awaits the tasks sequentially. An alternative to <see cref="Task.WhenAll(IEnumerable{Task})"/> and
+        /// <c>Nito.AsyncEx.TaskExtensions.WhenAll</c> when true multi-threaded asynchronicity is not desirable.
+        /// </summary>
+        /// <param name="source">A collection of items.</param>
+        /// <param name="asyncOperation">An <see langword="async"/> function to call on each item.</param>
+        /// <typeparam name="TItem">The type of the input collection's items.</typeparam>

```

Comment:
Unless I'm mistaken this is all identical to the method above and the only difference is the `<returns>`, right? You can use `<inheritdoc>` [like above](https://github.com/Lombiq/Helpful-Libraries/blob/1ebc4f82fe29ededc242da63fd93af2e1c11c150/Lombiq.HelpfulLibraries/Libraries/Utilities/EnumerableExtensions.cs#L28) and only override what's different.

Reply:
It also has no `TResult` parameter. I thought that's an issue but it actually doesn't seem to cause any problems, so I changed this.